src/bin/elementary/meson.build

```diff
@@ -177,6 +175,13 @@ if get_option('ethumb')
   elementary_test_src += ['test_thumb.c']
 endif
 
+if get_option('dbus')
+  elementary_test_src += [
+    'test_sys_notify.c',
+    'test_systray.c',
+  ]
```

Comment:
```suggestion
  elementary_test_src += files([
    'test_sys_notify.c',
    'test_systray.c',
  ])
```

Reply:
`elementary_test_src` don't use files, it is only a list of names

Reply:
? But it is used right there as a list of sources, line 181.

Reply:
Actually there's a plenty of missing `files(...)` in this file. _Oh lord.png_

Reply:
exactly, I didn't use files, because the variable isn't a `depend_files` and I don't know if it would be best to change it in this PR. Seems like a good thing for a phab ticket.

Reply:
`files` is not intended for a specific use case, it is intended to be used with file references. This _prevents_ bugs, so the one we had before was more of a consequence of not using it than a use case. Well, since I'm already messing with meson.build files, I could provide a patch for that later.

Reply:
There you go: https://phab.enlightenment.org/T8784.

Reply:
Nice just added something there.